apps/web/lib/daily-webhook/getBooking.ts

```diff
@@ -0,0 +1,54 @@
+import { HttpError } from "@calcom/lib/http-error";
+import logger from "@calcom/lib/logger";
+import { safeStringify } from "@calcom/lib/safeStringify";
+import prisma, { bookingMinimalSelect } from "@calcom/prisma";
+
+const log = logger.getSubLogger({ prefix: ["daily-video-webhook-handler"] });
+
+export const getBooking = async (bookingId: number) => {
+  const booking = await prisma.booking.findUniqueOrThrow({
+    where: {
+      id: bookingId,
+    },
+    select: {
+      ...bookingMinimalSelect,
+      uid: true,
+      location: true,
+      isRecorded: true,
+      eventTypeId: true,
+      eventType: {
+        select: {
+          teamId: true,
+          parentId: true,
+        },
+      },
+      user: {
+        select: {
+          id: true,
+          timeZone: true,
+          email: true,
+          name: true,
+          locale: true,
+          destinationCalendar: true,
+        },
+      },
+    },
```

Comment:
This should be a repository method in a new Booking repository. We could return all the scalars of Booking from that and for relations `eventType` and and `user`, we could maybe reuse the selects from https://github.com/calcom/cal.com/blob/48842e72a69575093f7b22c7869bcbb694f10385/packages/lib/server/repository/user.ts#L32 https://github.com/calcom/cal.com/blob/48842e72a69575093f7b22c7869bcbb694f10385/packages/lib/server/eventTypeSelect.ts#L3 ensuring no sensitive scalar value is sent.

Reply:
I created Booking Repository in my another PR https://github.com/calcom/cal.com/pull/15447. I'll update this code in the future.